featurebyte/models/feature.py

```diff
@@ -71,51 +72,39 @@ class BaseFeatureModel(QueryGraphMixin, FeatureByteCatalogBaseDocumentModel):
     It contains all the attributes that are shared between FeatureModel & TargetModel.
     """
 
-    dtype: DBVarType = Field(allow_mutation=False, default=DBVarType.UNKNOWN)
+    dtype: DBVarType = Field(default=DBVarType.UNKNOWN.value)
```

Comment:
Should this have `frozen=True`? Same question for some of the fields below that have `allow_mutation=False` before but without `frozen=True` after.